app/mailers/asset_notification.rb

```diff
@@ -1,22 +1,38 @@
 class AssetNotification < ActionMailer::Base
   default from: Alonetone.email
 
-  def upload_notification(asset, email, _sent_at = Time.now)
-    @track = asset.name
-    @description = asset.description
-    @name = asset.user.name
-    @user = asset.user
-    @title = asset.title.present? ? asset.title : "new track"
+  def upload_notification(asset, user, _sent_at = Time.now)
+    @user = user
+    @track = asset
     @play_link = play_link_for(asset)
-    @user_link = user_link_for(asset)
     @stop_following_link = stop_following_link
     @unsubscribe_link = unsubscribe_link
     @exclamation = %w[Sweet Yes Oooooh Alright Booya Yum Celebrate OMG].sample
-    mail subject: "[alonetone] '#{asset.user.name}' uploaded a new track!", to: email
+    mail subject: "[alonetone] '#{@user.name}' uploaded a new track!", to: @user.email
+  end
+
+  def upload_mass_notification(assets, user, _sent_at = Time.now)
+    @user = user
+    @tracks = generate_track_hash(assets)
```

Comment:
Since `Track` is a different model, can we rename this to `@assets` to prevent confusion?